File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kiem tra ngay thang nam/NGAY_THANG_NAM.cs
using System;
namespace Ngaythangnam
{
    public class Program
    {
        static void Main(string[]args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int d = int.Parse(input[0]);
            int m = int.Parse(input[1]);
            int y = int.Parse(input[2]);
            if((y >= 1900) && ((y % 400 == 0) || (y % 4 == 0)) && (y % 100 != 0) && m == 2 && d <= 29)
                Console.Write("YES");
            else  if((y >= 1900) && ((y % 400 != 0) || (y % 4 == 0)) && (y % 100!= 0)&& m==2 && d<=28)
                Console.Write("YES");
            else if((y >= 1900) && (m == 1 || m == 3 || m == 5 || m == 7 || m == 8 || m == 10 || m == 12) && d <= 31)
                Console.Write("YES");
            else if((y >= 1900) && (m == 4 || m == 6 || m == 9 || m == 11) && d <= 30)
                Console.Write("YES");
            else 
                Console.Write("NO");
        }
    }
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kiem tra ngay thang nam/NGAY_THANG_NAM.py
d, m, y = map(int, input().split())
if((y >= 1900) and ((y % 400 == 0) or (y % 4 == 0)) and (y % 100 != 0) and m == 2 and d <= 29):
    print("YES", end="")
elif((y >= 1900) and ((y % 400 != 0) or (y % 4 == 0)) and (y % 100!= 0)and m==2 and d<=28):
    print("YES", end="")
elif((y >= 1900) and (m == 1 or m == 3 or m == 5 or m == 7 or m == 8 or m == 10 or m == 12) and d <= 31):
    print("YES", end="")
elif((y >= 1900) and (m == 4 or m == 6 or m == 9 or m == 11) and d <= 30):
    print("YES",end= "")
else:
    print("NO",end= "")